.github/workflows/main.yaml

```diff
@@ -329,6 +332,19 @@ jobs:
         run: |
           make e2e-sync-import-tests
 
+      - name: Backup/Restore cluster tests
+        if: ${{ !cancelled() && steps.prepare-rancher.outcome != 'failure' && contains(inputs.tests_to_run, 'backup_restore') }}
+        env:
+          RANCHER_HOSTNAME: ${{ env.RANCHER_HOSTNAME }}
+          RANCHER_PASSWORD: ${{ env.RANCHER_PASSWORD }}
+          RANCHER_VERSION: ${{ inputs.rancher_version }}
+          BACKUP_OPERATOR_VERSION: ${{ inputs.backup_operator_version }}
+          CATTLE_TEST_CONFIG: ${{ github.workspace }}/cattle-config-provisioning.yaml
+          QASE_RUN_ID: ${{ steps.qase.outputs.qase_run_id }}
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+        run: |
+          make e2e-backup-restore-tests
```

Comment:
Could you export BACKUP_OPERATOR_VERSION to the GITHUB_SUMMARY? Our current workflow does not support a custom version, but still.

Reply:
Yes, its added in [summary](https://github.com/rancher/hosted-providers-e2e/actions/runs/12783135689/attempts/1#summary-35633769823)

Reply:
Oh, my bad. I did not notice it :sweat_smile: